variable/logical.cpp

```diff
@@ -15,27 +15,27 @@ using namespace scipp::core;
 namespace scipp::variable {
 
 static constexpr auto or_op_ = overloaded{
```

Comment:
@DavidFair I am rewriting this file (moving things to `core/element` as part of cleanup), so I will include you changes there.

Reply:
Thanks I'll drop this file from the PR :+1: